ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -30,6 +30,14 @@ let finally = Xapi_stdext_pervasives.Pervasiveext.finally
 
 let with_lock = Xapi_stdext_threads.Threadext.Mutex.execute
 
+let internal_error fmt =
```

Comment:
You have 2 copies of this, I assume because the 'error' function is different (it includes the module name).

Reply:
Indeed; otherwise you need to make sure it's logged under the correct name.